android/app/src/main/assets/microlocations.json

```diff
@@ -1,138 +1,138 @@
 [
     {
-        "floor": null, 
-        "id": 1, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 3, Level 3, Dalton Hall", 
+        "floor": null,
+        "id": 1,
+        "latitude": null,
+        "longitude": null,
+        "name": "Level 3, Level 3, Dalton Hall",
         "room": null
-    }, 
-    {
-        "floor": null, 
-        "id": 2, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 3, Lewis Lab", 
+    },
+    {
+        "floor": null,
+        "id": 2,
+        "latitude": null,
+        "longitude": null,
+        "name": "Level 3, Lewis Lab",
         "room": null
-    }, 
-    {
-        "floor": null, 
-        "id": 3, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 1, Digital Design Studio", 
+    },
+    {
+        "floor": null,
+        "id": 3,
+        "latitude": null,
+        "longitude": null,
+        "name": "Level 1, Digital Design Studio",
         "room": null
-    }, 
-    {
-        "floor": null, 
-        "id": 4, 
-        "latitude": null, 
-        "longitude": null, 
-        "name": "Level 1, Digital Design Studio", 
+    },
+    {
+        "floor": null,
+        "id": 4,
+        "latitude": null,
```

Comment:
Please update null values with geopoints in science centre singapore. Update these values on the github repo https://github.com/fossasia/open-event/tree/master/sample/FOSSASIA16 and create a PR. After this you wouldn't actually need to parse json. Yu just need to get the microlocations from the database and display them on the map.